tests/timeseries/test_logical_meter.py

```diff
@@ -176,14 +176,15 @@ async def test_grid_power_2(
         assert len(results) == 10
         assert self._equal_float_lists(results, meter_sums)
 
-    async def test_battery_and_pv_power(  # pylint: disable=too-many-locals
+    async def test_battery_pv_and_ev_power(  # pylint: disable=too-many-locals
```

Comment:
It would be better to test one thing in one test.
Could you add new test for ev_power only?